src/libs/ApiUtils.ts

```diff
@@ -52,22 +48,16 @@ function getApiRoot(request) {
 
 /**
  * Get the command url for the given request
- *
- * @param {Object} request
- * @param {String} request.command - the name of the API command
- * @param {Boolean} [request.shouldUseSecure]
- * @returns {String}
+ * @param - the name of the API command
```

Comment:
I don't think we still need this right?

Reply:
I think there is an agreement that we can leave params descriptions 
CC: @fabioh8010